values/decoding/time_decode.go

```diff
@@ -45,6 +45,7 @@ func (dec *timeDecoder) DecodeRaw(data []byte) (values.CloseableTimeValues, erro
 func newTimeIteratorFromMeta(
 	metaProto encodingpb.TimeMeta,
 	encodedBytes []byte,
+	scaleUp bool,
```

Comment:
Make this an enum perhaps? I'm a fan of more explicit parameters, e.g.,
```
type scaleMode int

const (
  noScale scaleMode = iota
  withScale
)
```
Then you can call this with `newTimeIterator(..., noScale)` which is more clear than `true` or `false`.